specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-15-preview/machineLearningServices.json

```
@@ -0,0 +1,5532 @@
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/services": {
+      "get": {
+        "tags": [
+          "MachineLearningServices"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "operationId": "MachineLearningService_ListByWorkspace",
+        "description": "Gets services in specified workspace.",
+        "x-ms-examples": {
+          "Get Services": {
+            "$ref": "./examples/getServices.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PaginationParameter"
+          },
+          {
+            "$ref": "#/parameters/ModelIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ModelNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceTagParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceTagsParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServicePropertiesParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceRunIdParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceExpandParameter"
+          },
+          {
+            "$ref": "#/parameters/WebServiceOrderByParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The response includes a paginated array of Machine Learning services and a URI to the next set of results, if any. For the more information the limits of the number of items in a resource group, see https://azure.microsoft.com/en-us/documentation/articles/azure-subscription-service-limits/.",
+            "schema": {
+              "$ref": "#/definitions/PaginatedServiceList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/services/{serviceName}": {
+      "get": {
+        "tags": [
+          "MachineLearningServices"
+        ],
+        "operationId": "MachineLearningService_Get",
```

Usually an operationId should have a plural noun as its receiver, for instance
```suggestion
        "operationId": "MachineLearningServices_Get",
```